contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -543,15 +543,22 @@
             roi index array.
 
             The values of this mask index the :math:`|Q|` value from the
-            the ``static_q_list`` field.
+            ``static_q_list`` field.
 
             The ``units`` attribute should be set to ``"au"``
             indicating arbitrary units.
             </doc>
           </field>
           <field name="static_q_list" type="NX_NUMBER" units="NX_PER_LENGTH" minOccurs="0">
             <doc>
-            1-D list of :math:`|Q|` values, 1 for each roi.
+            1-D list of :math:`|Q|` values, one for each roi.
```

Comment:
Should this `doc` field describe the same options as in its dynamic counterpart?
* Q(r)
* Q(r), phi
* H, K, L
* qx, qy, qz
* x, y pixels

Reply:
Good catch!  You are correct about the inconsistent use of `Q` and `q`.  In small-angle (which is used together sometimes with XPCS), $Q$ has a different meaning than reciprocal space vector; it represents the total scattered intensity, integrated over all $\vec{q}$.  I will suggest a revision as part of this PR.  Should that discussion become distracting to the intent of this PR (as stated in the title), I suggest moving such comment and revision to a separate issue.

Reply:
@sureshnaps, @AZjk, @qzhang234, @ambarb, @AbbyGi: Can you comment if the options and other documentation for `dynamic_q_list` should be copied to `static_q_list`?

Reply:
@qzhang234 Thanks for the gathering input from APS folks and sharing the consensus here!

I think that makes sense to fold these into single lists--one for static and one for dynamic. @ambarb: Is this what you were thinking also?

Reply:
Since many of those columns (`q`, `phi`, `qx`, `qy`, `qz`, `h`, `k`, `l`, etc.) would be empty if we insist that all of them be present...@ambarb had suggested that we could instead add a [@storage_mode](https://manual.nexusformat.org/classes/contributed_definitions/NXxpcs.html#nxxpcs-entry-twotime-two-time-corr-func-storage-mode-attribute) attribute. 

This would be similar to how it is used in the `NXxpcs::entry::twotime` group, and it could have defined values, such as `q_only`, `q_phi`, `qxyz`, `hkl`, `xy`, or whatever makes sense here.  Effectively this would act as a [tagged union](https://en.wikipedia.org/wiki/Tagged_union) discriminator for the q_lists.  I think this sounds useful.

This could be split out into a separate PR, but it seemed relevant to bring it up here.  What do others think?

Reply:
one thing not yet resolved is how the `static_q_list` is envisioned to be used.  I imagine that the list correlates an intensity list so that one may plot I(Q).  @qzhang234  can you confirm?  

The exact wording in our current NXxpcs definition for static and dynamic q-labels is:

**"Dynamic” represents quantities directly related to XPCS and NXxcps/entry/data and NXxpcs/entry/two_time.

“Static” refers to finer binning used for computation not strictly used for the final XPCS results. Implementation of _static_ binning is left for individual libraries to document. We encourage usage of [NXcanSAS](https://manual.nexusformat.org/classes/applications/NXcanSAS.html#nxcansas) to represent standard SAXS results or development of new NeXus definitions for GI-SAXS or other reciprocal space intensity mapping.**

I was wrong, there is a `static_roi_map` so this part is handled, but I see absolutely no data to plot against the x-variables (`static_q_list` or `static_phi_list`).  I believe this is because we left it to others to document as the preference was to use NXcanSAS.  I am fine with what the community wants to do in this regard (I(Q) fully inside custom NXxpcs  versus NXcanSAS), but if the choice is fully inside NXxcps, it seems better to fully document beyond just adding "static_phi_list" and the other discussed changes.

Reply:
https://github.com/nexusformat/definitions/pull/1314#discussion_r1376543362. should have been in this thread, not where it is at the bottom of this page

Reply:
> Since many of those columns (`q`, `phi`, `qx`, `qy`, `qz`, `h`, `k`, `l`, etc.) would be empty if we insist that all of them be present...@ambarb had suggested that we could instead add a [@storage_mode](https://manual.nexusformat.org/classes/contributed_definitions/NXxpcs.html#nxxpcs-entry-twotime-two-time-corr-func-storage-mode-attribute) attribute.
> 
> This would be similar to how it is used in the `NXxpcs::entry::twotime` group, and it could have defined values, such as `q_only`, `q_phi`, `qxyz`, `hkl`, `xy`, or whatever makes sense here. Effectively this would act as a [tagged union](https://en.wikipedia.org/wiki/Tagged_union) discriminator for the q_lists. I think this sounds useful.
> 
> This could be split out into a separate PR, but it seemed relevant to bring it up here. What do others think?

https://github.com/nexusformat/definitions/pull/1314#discussion_r1376543362. should have been in this thread, not where it is at the bottom of this page